lib/api_me/pagination.rb

```diff
@@ -1,39 +1,56 @@
+# frozen_string_literal: true
```

Comment:
Been adding this when touching files too, is this a new rubocop addition? It seems new, but it makes sense.